pyproject.toml

```diff
@@ -556,7 +556,8 @@ disable = [
   "fixme",
   "consider-using-assignment-expr",
   "logging-fstring-interpolation",
-  "consider-using-any-or-all"
+  "consider-using-any-or-all",
+  "too-many-instance-attributes",
```

Comment:
Don't disable the checker, rather increase the max-attributes. 
https://docs.pylint.org/features.html

Reply:
Updated `max-attributes = 13`